src/v2/Apps/Onboarding/Views/OnboardingQuestionTwo.tsx

```diff
@@ -0,0 +1,99 @@
+import {
+  Flex,
+  Spacer,
+  Text,
+  Join,
+  Box,
+  CheckCircleFillIcon,
+  Button,
+} from "@artsy/palette"
+import { FC } from "react"
+import {
+  OPTION_ARTWORKS_FROM_TRENDING_ARTISTS,
+  OPTION_AUCTION_HIGHLIGHTS_FROM_THIS_WEEK,
+  OPTION_A_CURATED_SELECTION_OF_ARTWORKS,
+  OPTION_COLLECTING_ART_THAT_MOVES_ME,
+  OPTION_CREATE_AN_ART_WISHLIST,
+  OPTION_CURRENTLY_TRENDING_ARTISTS,
+  OPTION_FINDING_MY_NEXT_GREAT_INVESTMENT,
+  OPTION_FIND_NEW_ART_FROM_ARTISTS_THAT_I_COLLECT,
+  OPTION_FOLLOW_ARTISTS_I_COLLECT,
+  OPTION_FOLLOW_GALLERIES_I_LOVE,
+  OPTION_GROW_MY_TASTE_IN_ART,
+  OPTION_KEEPING_TRACK_OF_ART_I_AM_INTERESTED_IN,
+  OPTION_KEEP_TRACK_OF_ARTIST_CAREER,
+  OPTION_POPULAR_AUCTION_LOTS,
+} from "../config"
+import { useOnboardingContext } from "../useOnboardingContext"
+
+export const OnboardingQuestionTwo: FC = () => {
+  const { answers, setAnswers, setContext, next } = useOnboardingContext()
+
+  const options = QUESTION_2[answers[0]!] as string[]
```

Comment:
😅

Reply:
Things like this should be abstracted a bit if possible. A future dev will have a hard time understanding how this was arrived at.